participants/MedicalImageCloudSystem.md

```diff
@@ -0,0 +1,105 @@
+# Medical Image Cloud System
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Please tell us your project's name and the Filecoin Slack handles for your teammates.*
+
+Project Name: Medical Image Cloud System
+
+Slack user: anyi
+
+## (1) Brief Project Description
+
+*Add your project's description here.* 
+
+The Medical image cloud system (MICS)is to store medical images and support image data retrieval for Web service.these images stored on IPFS are from many local alliance hospitals, and serval data copies are saved in multiple IPFS nodes to ensure data security in case of data lost. Access is restricted by permission contronl. only authorized doctors have rights to search and share image data,they can view the image data and other information by web browsers. 
```

Comment:
Just clarifying that the data will be transferred to the Filecoin network from IPFS.